eZ/Publish/Core/MVC/Symfony/Matcher/Tests/ContentBased/BaseTest.php

```diff
@@ -14,7 +14,7 @@
 use eZ\Publish\Core\Repository\Helper\RoleDomainMapper;
```

Comment:
Could you also remove `File containing the BaseTest class.` like in other files?

Reply:
> Could you also remove `File containing the BaseTest class.` like in other files?

Aligned via 7856160956.